atc/api/policychecker/handler.go

```diff
@@ -0,0 +1,49 @@
+package policychecker
+
+import (
+	"code.cloudfoundry.org/lager"
+	"context"
+	"net/http"
+
+	"github.com/concourse/concourse/atc/api/accessor"
+	"github.com/concourse/concourse/atc/policy"
+)
+
+func NewHandler(
+	logger lager.Logger,
+	handler http.Handler,
+	action string,
+	policyChecker policy.PreChecker,
+) http.Handler {
+	return policyCheckerHandler{
+		handler:       handler,
+		action:        action,
+		policyChecker: policyChecker,
+	}
+}
+
+type policyCheckerHandler struct {
```

Comment:
Another English nit - if something is called a `thingHandler` I would expect it to handle `thing`s. So I would expect a `policyCheckerHandler` to handle policycheckers. But this struct is called a `Handler` because it fulfills the `http.Handler` interface!
```suggestion
type policyCheckingHandler struct {
```